test/helpers.js

```diff
@@ -1,16 +1,5 @@
-/**
-    helpers for tests
-*/
 import { Utils } from "../lib/utils.js";
 import { assert } from "chai";
-
-beforeEach(async () => {
-  global.web3 = Utils.getWeb3();
-  global.assert = assert;
-  global.accounts = [];
-  await etherForEveryone();
```

Comment:
we dont need this anymore?

Reply:
It existed in two places, was duplicated.